pkg/sdn/api/validation/validation.go

```diff
@@ -15,21 +15,25 @@ func ValidateClusterNetwork(clusterNet *sdnapi.ClusterNetwork) fielderrors.Valid
 	allErrs := fielderrors.ValidationErrorList{}
 	allErrs = append(allErrs, validation.ValidateObjectMeta(&clusterNet.ObjectMeta, false, oapi.MinimalNameRequirements).Prefix("metadata")...)
 
-	_, ipnet, err := net.ParseCIDR(clusterNet.Network)
+	_, clusterIPNet, err := net.ParseCIDR(clusterNet.Network)
 	if err != nil {
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("network", clusterNet.Network, err.Error()))
 	} else {
-		ones, bitSize := ipnet.Mask.Size()
+		ones, bitSize := clusterIPNet.Mask.Size()
 		if (bitSize - ones) <= clusterNet.HostSubnetLength {
 			allErrs = append(allErrs, fielderrors.NewFieldInvalid("hostSubnetLength", clusterNet.HostSubnetLength, "subnet length is greater than cluster Mask"))
 		}
 	}
 
-	_, _, err = net.ParseCIDR(clusterNet.ServiceNetwork)
+	serviceIP, _, err := net.ParseCIDR(clusterNet.ServiceNetwork)
 	if err != nil {
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("serviceNetwork", clusterNet.ServiceNetwork, err.Error()))
 	}
 
+	if clusterIPNet.Contains(serviceIP) {
```

Comment:
Can you check with the install team to make sure the same validation is done when setting up the configuration?